### PR TITLE
[Update] Modularize symbology category

### PR DESCRIPTION
**Pull request by Luiy0**
### Description

Modularize and remove deprecated code (colors) from samples in the `symbology` category.

Checklist:
- [ ] Set target branch to main (current release) or v.next (next release)
- [ ] Request a reviewer
- [ ] Assign a reviewer
- [ ] Tag reviewer in comment

### CODE REVIEW

File: symbology/picture-marker-symbol/build.gradle [hunk 1]
```diff
@@ -1,17 +1,22 @@
...
 
 ext {
-    arcgisVersion = '100.15.0'
+    arcgisVersion = '200.0.0-3672'
 }
 
 javafx {
     version = "17.0.2"
-    modules = [ 'javafx.controls', 'javafx.swing' ]
+    modules = [ 'javafx.controls', 'javafx.graphics', 'javafx.fxml', 'javafx.web', 'javafx.media', 'javafx.swing' ]
```
Comment (jenmerritt) [inline symbology/picture-marker-symbol/build.gradle]: I think we should refactor this sample to get rid of `javafx.swing` as it's a bit of an unnecessary use of it too. Looking at other samples, the key is to show:
1. creating a picture marker symbol from a URL to a service (which we do)
2. creating a PMS from an `Image` (which we do)
3. creating a PMS from a URL to a file path - which we're doing in a very convoluted way. I think we should do something like:

```
      URL res = getClass().getClassLoader().getResource("orange_symbol.png");
      File file = Paths.get(res.toURI()).toFile();
      orangeSymbol = new PictureMarkerSymbol(file.getAbsolutePath());
      placePictureMarkerSymbol(orangeSymbol, leftPoint);
```

And removing the whole bit `saveResourceToExternalStorage`.
Give me a shout if this isn't clear

File: symbology/picture-marker-symbol/src/main/java/module-info.java [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  // require ArcGIS Runtime module
+  requires com.esri.arcgisruntime;
+
+  // require JavaFX modules that the application uses
+  requires javafx.graphics;
+  requires javafx.controls;
+
+  // require other modules that the application uses
+  requires java.desktop;
+  requires javafx.swing;
```
Comment (jenmerritt) [inline symbology/picture-marker-symbol/src/main/java/module-info.java]:
```suggestion
```

Should be able to lose these after the changes

**Review by jenmerritt — COMMENTED** [review]
@Luiy0 few comments below, thanks

**Comment by Luiy0** [comment]
Hi @jenmerritt, sample is working correctly after implementing the suggested changes.

**Review by jenmerritt — APPROVED** [review]
Nice @Luiy0 , I think that's improved the Picture Marker Symbol sample too.

**Comment by jenmerritt** [comment]
Updated to improve how Picture Marker Symbol is accessed via Sample Viewer. Good to merge when you're ready @Luiy0

**Pull request merged**